app/src/main/java/com/p2p/presentation/truco/TrucoPlayFor4Fragment.kt

```diff
@@ -121,6 +124,22 @@ class TrucoPlayFor4Fragment : TrucoFragment<FragmentPlayTrucoFor4Binding>() {
         }
     }
 
+    private fun getHandPlayerIcon(playerPosition: TrucoPlayerPosition) = when (playerPosition) {
+        TrucoPlayerPosition.FRONT -> gameBinding.frontPlayerHand
+        TrucoPlayerPosition.LEFT -> gameBinding.leftPlayerHand
+        TrucoPlayerPosition.RIGHT -> gameBinding.rightPlayerHand
+        TrucoPlayerPosition.MY_SELF -> null
+    }
+
+    private fun setHandPlayerIcon(handPlayerIcon: ImageView?) {
+        gameBinding.frontPlayerHand.visibility = View.INVISIBLE
+        gameBinding.leftPlayerHand.visibility = View.INVISIBLE
+        gameBinding.rightPlayerHand.visibility = View.INVISIBLE
+        if (handPlayerIcon != null) {
+            handPlayerIcon.visibility = View.VISIBLE
+        }
```

Comment:
```suggestion
        listOf(gameBinding.frontPlayerHand, gameBinding.leftPlayerHand, gameBinding.rightPlayerHand).forEach {
            is.isVisible = it == handPlayerIcon
        }
```
Y la lista en realidad la podrías tener declarada al principio como el resto de las cosas para simplificar 😅

Reply:
Hermoso!